tests/nativeapp/test_event_sharing.py

```diff
@@ -550,13 +597,27 @@ def test_event_sharing_disabled_but_we_add_event_sharing_flag_in_project_definit
         console=mock_console,
     )
 
-    assert mock_execute_query.mock_calls == mock_execute_query_expected
-    mock_console.warning.assert_called_with(
-        "WARNING: Same-account event sharing is not enabled in your account, therefore, application telemetry section will be ignored."
-    )
+    assert [
+        *mock_execute_query.mock_calls,
+        *mock_sql_facade_upgrade_application.mock_calls,
+        *mock_sql_facade_create_application.mock_calls,
+    ] == expected
+
+    assert mock_console.warning.mock_calls == [
+        mock.call(
+            "WARNING: Same-account event sharing is not enabled in your account, therefore, application telemetry section will be ignored."
+        ),
+        mock.call(
+            DEFAULT_UPGRADE_SUCCESS_MESSAGE
+            if is_upgrade
+            else DEFAULT_CREATE_SUCCESS_MESSAGE
```

Comment:
for simplicity, you can have the same message since this is within the same test. We already check the mocks for upgrade/create, so it's hard to mess this up.

Reply:
For sure, updated here: https://github.com/snowflakedb/snowflake-cli/pull/1870/commits/17f761b0ef35025d5e2f07f9f0e6ceb47d3068f6